front-end/src/components/Reactionbar/query.ts

```diff
@@ -8,7 +8,7 @@ export const postReactionFields = gql`
     fragment postReactionFields on post_reactions {
         id
         reacting_user {
-            id
+            username
```

Comment:
We have had problems in the past with queries not containing the id. I solved it #99, but didn't log the error unfortunately. So we can keep the id in there.

Reply:
done